pkg/controller/storagecluster/storageclasses.go

```diff
@@ -84,14 +84,9 @@ func (r *ReconcileStorageCluster) newOBCStorageClass(initData *ocsv1.StorageClus
 		Parameters: map[string]string{
 			"objectStoreNamespace": initData.Namespace,
 			"region":               "us-east-1",
+			"objectStoreName":      generateNameForCephObjectStore(initData),
```

Comment:
So now we're generating this for both external and converged modes? Did something that drastic really change in the OBC StorageClass?

Reply:
A minor change in the OBC StorageClass' `Parameter map[string]string` now takes `objectStoreName`, instead of direct `endpoint`.

Reply:
I'd hardly call this minor. :) Also, you didn't answer my question: this is for both external and converged modes? How does it get used in converged mode, and how did it work before without this?

Reply:
Oops! missed the question mark (read the first line as a statement not as question)...
Yes, we are generating the CephObjectStore for both external and converged mode... But the creation of `CephObjectStore` is different in each mode,
a. Converged mode - we use ensure function to create CephObjectStore
b. External mode - we create it when `ceph-rgw` storageclass is available with the endpoint data in it.

@leseb ^^^